rudder-web/src/main/webapp/secure/administration/policyServerManagement.html

```diff
@@ -118,6 +118,31 @@
           </div>
         </div>
         <hr class="spacer" />
+        <div class="deca">
+          <h3>Protocol</h3>
+          <div class="lift:administration.PropertiesManagement.networkProtocolSection" id="networkProtocolForm">
+            <form class="lift:form.ajax">
+              <div class="wbBaseField">
+                    <label for="reportProtocol" class="threeCol textight" style="font-weight:bold;width: 50%;">
+                      <span >Use UDP in place of TCP for syslog communication between node and servers:
```

Comment:
I think we need to have the sentence in the other way if UDP is the default:
"Use TCP in place of UDP (default) for syslog communication between node and servers:"